utils/vpr_pbtype_arch_wrapper.py

```diff
@@ -1,4 +1,8 @@
 #!/usr/bin/env python3
+"""
+Tool for generate an arch.xml file which includes pb_type.xml and model.xml
+files for testing with Verilog to Routing.
```

Comment:
Maybe worth mentioning the CMake function name that uses it?

Reply:
Done.